### PR TITLE
chore: remove repetitive words

### PR DESCRIPTION
**Pull request by fanqiaojun**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
I'm okay calling this a trivial change.  If you agree, please amend your commit message to include a line "CLA: trivial" down the bottom.  Otherwise, we'll need to you fill in an [ICLA](https://www.openssl.org/policies/cla.html).

**Review by t-j-h — APPROVED** [review]

File: demos/README.txt [hunk 1]
```diff
@@ -3,6 +3,10 @@ OpenSSL Demonstration Applications
 This folder contains source code that demonstrates the proper use of the OpenSSL
 library API.
 
+Note: Makefiles are provided in the demo subfolders to demonstrate how to build
+them, but are not frequently used.  Configure openssl with enable-demos to build
+them automatically through the perl based build system
+
```
Comment (t8m) [inline demos/README.txt:9]: This is an unrelated change, can you please drop it from the PR?

**Review by t8m — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
Merged to the master branch. Thank you for your contribution.
